tests/e2e/nvidia-container-cli_test.go

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"text/template"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	installDockerTemplate = `docker exec -u root {{.ContainerName}} bash -c '
```

Comment:
Why not have the script be the "template" directly? i.e. not include the `docker exec` part? In most cases this would mean that the script is used as is and does not even require templating.

Reply:
Ok, I got what I was doing wrong, now is not a template